src/utils/url.ts

```diff
@@ -0,0 +1,70 @@
+import { NotSupportedPlatformError } from "../main/exceptions/not-supported-platform";
+
+const MACOS_FILE_NAME = "macOS.tar.gz";
+const LINUX_FILE_NAME = "Linux.tar.gz";
+const WINDOWS_FILE_NAME = "Windows.zip";
+
+export interface IDownloadUrls {
+  launcher: string;
+  player: string;
+}
+
+export function getDownloadUrls(
```

Comment:
If we are going to fetch the project version over the network, making this function async would be reasonable.

Reply:
I think these url util will remain pure in the future